src/app/layout.tsx

```diff
@@ -15,7 +20,19 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body>
-        <ClientProviders>{children}</ClientProviders>
+        <Grid className="grid min-h-screen grid-rows-[auto,1fr,auto]">
+          <div className="row-start-1 row-end-2">
+            <Header />
+          </div>
+          <div className="row-start-2 row-end-3 overflow-auto">
+            <div className="mx-auto max-w-7xl px-4 py-8">
+              <ClientProviders>{children}</ClientProviders>
+            </div>
+          </div>
+          <div className="row-start-3 row-end-4">
```

Comment:
I'm also still getting used to radix-ui, but let's try to use its components as much as possible. I some of these divs we could probably use one of these components instead:

![CleanShot 2024-11-08 at 12 50 58@2x](https://github.com/user-attachments/assets/d870b1e3-d616-4c6b-bfc8-b08492d0ba69)